src/harness/vfs.ts

```diff
@@ -1201,6 +1350,24 @@ namespace vfs {
         }
     }
 
+    /** Removes a directory in a `FileSet` */
+    export class Rmdir {
+        private _rmdirBrand?: never; // brand necessary for proper type guards
+    }
+
+    /** Unlinks a file in a `FileSet` */
+    export class Unlink {
+        private _unlinkBrand?: never; // brand necessary for proper type guards
+    }
+
+    // prevents compiler errors due to unused brand properties

```

Comment:
Don't we have some `_unlinkBrand?!` or `["_unlinkBrand"]` syntax variant that should more cleanly suppress that error? Or is that just definite assignment?